include/spm.rst

```diff
@@ -50,6 +50,9 @@ Peripherals configured as Non-Secure
    * TWIM2
    * UARTE0, UARTE1
 
+If you want to use software breakpoints when stepping through your code, you would have to disable the lock bit on permissions.
```

Comment:
on flash and sram permissions, right? Cause, the peripheral permissions are not affected

Reply:
That's correct, we only need to be able to modify the permissions set to insert code in flash or ram for setting software breakpoints.